src/table_printer/table_printer.hpp

```diff
@@ -228,8 +234,22 @@ class table_printer {
    */
   void set_output_stream(std::ostream& out_stream) {
     alt_output_stream = &out_stream;
-  }  
-  
+  }
+
+  static void _os_log_value(size_t column_index, unsigned long long value);
+  static void _os_log_value(size_t column_index, unsigned long value);
+  static void _os_log_value(size_t column_index, unsigned int value);
+  static void _os_log_value(size_t column_index, long long value);
+  static void _os_log_value(size_t column_index, long value);
+  static void _os_log_value(size_t column_index, int value);
+  static void _os_log_value(size_t column_index, double value);
+  static void _os_log_value(size_t column_index, float value);
+  void _os_log_value(size_t column_index, const progress_time& value) const;
+  static void _os_log_value(size_t column_index, char* value);
+  static void _os_log_value(size_t column_index, const char* value);
```

Comment:
Nit: the `const char*` override should supersede the `char*` override entirely, right?

Reply:
Thanks, I'll try removing the non-const version.